home-application/src/main/java/com/softserveinc/ita/homeproject/application/apiservice/NewsApiServiceImpl.java

```diff
@@ -118,4 +118,4 @@ public Response updateNews(Long id, UpdateNews updateNews) {
         return Response.ok().entity(response).build();
     }
 
-}
+}
```

Comment:
Please do not forget to add a newline at the end of the file.

Reply:
Ok, will try to remember that.

Reply:
Please add newline at the end of the file.